analysis/filter/MergeFilter/src/main/kotlin/de/maibornwolff/codecharta/filter/mergefilter/LargeMerge.kt

```diff
@@ -0,0 +1,54 @@
+package de.maibornwolff.codecharta.filter.mergefilter
+
+import de.maibornwolff.codecharta.model.BlacklistItem
+import de.maibornwolff.codecharta.model.Edge
+import de.maibornwolff.codecharta.model.Node
+import de.maibornwolff.codecharta.model.NodeType
+import de.maibornwolff.codecharta.model.Project
+
+class LargeMerge {
+    companion object {
+        fun packageProjectInto(project: Project, prefix: String): Project {
```

Comment:
maybe wrapProjectInFolder is a better name

Reply:
I agree, good name